Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -489,6 +489,35 @@ def audit_conflicts
     EOS
   end
 
+  def audit_keg_only_style
+    return unless @strict
+    return unless formula.keg_only?
+
+    # This should catch most things, except awkwardness like
+    # LibreSSL or OpenSSL, which we can manually catch.
+    formula_name = formula.name.sub(/^./, &:upcase)
```

Comment:
How about doing a case insensitive removal of the formula name from the caveats?